### PR TITLE
OP-312 - telemetry module integrations

### PR DESCRIPTION
**Pull request by AndreiDodu**
- merged develop 
- changed FreeGeoIPJSON -> GeoIpLookup service

### CODE REVIEW

File: src/main/java/org/isf/generaldata/GeneralData.java [hunk 1]
```diff
@@ -143,14 +146,15 @@ public static void reset() {
...
+		USERSLISTLOGIN = myGetProperty("USERSLISTLOGIN", DEFAULT_USERSLISTLOGIN);
 		LANGUAGE = myGetProperty("LANGUAGE", DEFAULT_LANGUAGE);
 		AUTOMATICLOT_IN = myGetProperty("AUTOMATICLOT_IN", DEFAULT_AUTOMATICLOT_IN);
 		AUTOMATICLOT_OUT = myGetProperty("AUTOMATICLOT_OUT", DEFAULT_AUTOMATICLOT_OUT);
 		AUTOMATICLOTWARD_TOWARD = myGetProperty("AUTOMATICLOTWARD_TOWARD", DEFAULT_AUTOMATICLOTWARD_TOWARD);
 		LOTWITHCOST = myGetProperty("LOTWITHCOST", DEFAULT_LOTWITHCOST);
 		PATIENTSHEET = myGetProperty("PATIENTSHEET", DEFAULT_PATIENTSHEET);
 		VISITSHEET = myGetProperty("VISITSHEET", DEFAULT_VISITSHEET);
-		EXAMINATIONCHART = myGetProperty("EXAMINATIONCHART", DEFAULT_EXAMINATIONCHART);
+		EXAMINATIONCHART =myGetProperty("EXAMINATIONCHART", DEFAULT_EXAMINATIONCHART);
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/generaldata/GeneralData.java:157]: why was the space removed after the equals sign?

File: src/main/java/org/isf/telemetry/daemon/TelemetryDaemon.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ public TelemetryDaemon() {
 	public void run() {
 		while (running) {
 			LOGGER.info("Telemetry module running ({})...", updateSettingsCounter);
-			boolean isSendingMessageServiceActive = settings.getActive().booleanValue();
+			boolean isSendingMessageServiceActive = settings!=null && settings.getActive() != null ? settings.getActive().booleanValue() : false;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/telemetry/daemon/TelemetryDaemon.java:68]: there should be spaces around `!=`

File: src/main/java/org/isf/generaldata/GeneralData.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/generaldata/GeneralData.java]: Done, thanks!

File: src/main/java/org/isf/telemetry/daemon/TelemetryDaemon.java (hunk 2 above)
Comment (mwithi) [reply src/main/java/org/isf/telemetry/daemon/TelemetryDaemon.java]: Done, thanks!

**Pull request merged**